main/boost/APKBUILD

```diff
@@ -1,14 +1,14 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=boost
-pkgver=1.67.0
+pkgver=1.68.0
 _pkgver="${pkgver//./_}"
-pkgrel=1
+pkgrel=0
 pkgdesc="Free peer-reviewed portable C++ source libraries"
 url="http://www.boost.org/"
 arch="all"
 license="custom"
-depends=""
-depends_dev="linux-headers"
+depends="$pkgname-libs"
+depends_dev="$pkgname $pkgname-static=$pkgver-r$pkgrel linux-headers"
```

Comment:
why the hard dependency on the `boost-static` package? If it really is a hard dependency (eg. not optional) then we should just include the static libs in the -dev package.

If it is optional (and you can build with only dynamic libs) then the `boost-dev` package should not depend on `boost-static`

Reply:
That's fair. I'll remove that dependency.